README.md

```diff
@@ -1196,6 +1196,39 @@ peered with.
 
 `peer_vpc` defines the identifier or name of the remote VPC.
 
+### Datasource Service Component
```

Comment:
This could start by saying something like

Service components can be defined to get the connection info for specific service. Services may support multiple different access routes (VPC peering and public access), have additional components or support multiple authenticatoin methods. Each of these may be represented by different DNS name or TCP port and the specific component to match can be selected by specifying appropriate filters as shown below.